src/template-tags/tickets.php

```diff
@@ -10,6 +10,46 @@
 	exit( '-1' );
 }
 
+if ( ! function_exists( 'tribe_tickets_parent_post' ) ) {
+	/**
+	 * Returns the current post object that can have tickets attached to it
+	 *
+	 * Optionally the post object or ID of a post can be passed in and,
```

Comment:
Might it be clearer to write, _the post object or ID of a **ticket** post can be passed in and, again, the **parent (event)** post object will be returned if possible._

Though I'm pretty sure lots of us will get the gist, it's slightly ambiguous as currently worded.